docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+privileges on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-reader-role.jpg["Creating a role in Kibana"]
+
+Create a `metricbeat_writer` role that has `manage_index_templates` and `monitor` 
+cluster privileges, as well as `write`, `delete`, and `create_index` privileges
+on the `metricbeat-*` indices:
+
+// TBD: The Beats docs don't mention the need for the delete privilege, whereas 
+// the Logstash docs do. Which should be used in this case?
+
+[role="screenshot"]
+image::security/images/create-writer-role.jpg["Creating another role in Kibana"]
+
+Now go back to the *Management / Security / Users* page and assign these roles 
+to the appropriate users. Assign the `metricbeat_reader` role to your personal 
+user.  Assign the `metricbeat_writer` role to the `metricbeat_internal` user.
+
+The list of users should now contain all of the built-in users as well as the 
+two you created. It should also show the appropriate roles for your users:
+
+[role="screenshot"]
+image::security/images/management-users.jpg["User management screenshot in Kibana"]
+
+[float]
+[[gs-logstash-security]]
+=== Add users in {ls}
+
+In order for {ls} to send data successfully to {es}, you must configure its 
+authentication credentials. You must configure credentials separately for each 
+of the {es} plugins in your {ls} configuration file.  For example, update the `demo-metrics-pipeline.conf` file in your {ls} directory:
+
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+filter {
+  if [system][process] {
+    if [system][process][cmdline] {
+      grok {
+        match => {
+          "[system][process][cmdline]" => "^%{PATH:[system][process][cmdline_path]}"
+        }
+        remove_field => "[system][process][cmdline]"
+      }
+    }
+  }
+}
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
+    user => metricbeat_internal <1>
+    password => your_password <2>
+  }
+}
+----
+<1> Specify the `metricbeat_internal` user that you created earlier in this tutorial. 
+<2> Specify the password that you chose for this user ID.
+
+<<gs-start-logstash,Start {ls}>>. 
+
+[float]
+[[gs-metricbeat-security]]
+=== Add users in {metricbeat}
+
+If you were connecting directly from {metricbeat} to {es}, you would need to 
+configure authentication credentials for the {es} output in the {metricbeat} 
+configuration file. 
+
+In <<get-started-elastic-stack>>, however, you configured {metricbeat} to send 
+the data to {ls} for additional parsing, so no extra settings are required in 
+{metricbeat}. 
+
+<<gs-start-metricbeat,Start {metricbeat}>>. 
+
+[float]
+[[gs-view-security]]
+=== View system metrics in {kib}
+
+Log in to {kib} with the user ID that has `metricbeat_reader` role (for example, 
+`jdoe`). 
+
+This user should have authority to visualize the system metrics (for example, on 
```

Comment:
I'd change the wording here because readers might wonder whether there's an additional step required. If they've followed the steps in the tutorial, the user will have the authority to visualize system metrics, right?